frontend/apps/reader/modules/readerview.lua

```diff
@@ -695,7 +695,8 @@ function ReaderView:recalculate()
             -- start from right of page_area
             self.visible_area.x = self.page_area.x + self.page_area.w - self.visible_area.w
         end
-        if self.document.configurable.zoom_direction >= 2 and self.document.configurable.zoom_direction <= 5 then -- zoom_bottom_to_top
+        -- Check if we are in zoom_bottom_to_top
+        if self.document.configurable.zoom_direction and self.document.configurable.zoom_direction >= 2 and self.document.configurable.zoom_direction <= 5 then
```

Comment:
This would be nil before ReaderZooming is loaded or some such?

Reply:
Probably configurable.zoom_direction is never nil for mupdf engine but can be nil for Image view or Picture Document engines, maybe not even requiring this module or such at all.